src/main/webapp/js/messages-loader.js

```diff
@@ -84,6 +84,7 @@ function createCinemaOptions() {
         cinemaSelect.appendChild(option);
```

Comment:
Just wondering what's the benefit of using async/await vs putting the `cinemaSelect.value = cinemaKey;` line here in this callback instead?

Reply:
I think you are right. I didn't think of it when I was implemented. Changed to put in callback for better consistency.

Reply:
@jiangyue12392 not sure from where and how but line 76 has a missing ';' at the end.

Reply:
I think we somehow missed it in previous reviews. I will correct it together here then